doc/refman/RefMan-tac.tex

```diff
@@ -1334,13 +1334,17 @@ \subsection{\tt assert ( {\ident} :\ {\form} )}
       quantifications or non-dependent implications) are instantiated
       by concrete terms coming either from arguments \term$_1$
       $\ldots$ \term$_n$ or from a bindings list (see
-      Section~\ref{Binding-list} for more about bindings lists). In the
-      second form, all instantiation elements must be given, whereas
-      in the first form the application to \term$_1$ {\ldots}
+      Section~\ref{Binding-list} for more about bindings lists). 
+      In the first form the application to \term$_1$ {\ldots}
       \term$_n$ can be partial. The first form is equivalent to
       {\tt assert (\ident' := {\ident} {\term$_1$} \dots\ \term$_n$);
            clear \ident; rename \ident' into \ident}.
```

Comment:
It seems to me that this is not based on the latest doc (I made some changes to the doc of `specialize` very recently). Could you rebase on trunk?